tests/integration/targets/dcnm_fabric/tests/dcnm_fabric_replaced_save_deploy.yaml

```diff
@@ -145,7 +149,10 @@
 #                 },
 #                 "MESSAGE": "OK",
 #                 "METHOD": "POST",
-#                 "REQUEST_PATH": "https://172.22.150.244:443/appcenter/cisco/ndfc/api/v1/rest/control/fabrics/LAN_CLASSIC_Fabric/LAN_Classic",
+#                 "REQUEST_PATH": (
```

Comment:
We need to revert this change.  This breaks up a single API URI into two different strings which is not what we want